### ISSUE TITLE
Expand live documentation

### ISSUE THREAD

**Issue by expez** [expez/superstring#10]
Only a few of the functions are documented, we should add the rest.

**Comment by viebel** [comment]
I just noticed that it is super-easy to have the `docs` folder of your `master` branch published automatically by github.

Open the settings of your repo and in the the `Github Pages` section, you have this option:

![image](https://cloud.githubusercontent.com/assets/955710/18227180/42c53e40-7224-11e6-8bd8-11b1231ea44b.png)

**Comment by expez** [comment]
Cool, but how will that work with the existing api documentation? http://expez.github.io/superstring/doc/superstring.core.html  Kinda looks like that lives at the same place...

**Comment by viebel** [comment]
No worries, I've been working on integrating KLIPSE with codox.

I've opened a Pull request on Codox:  https://github.com/weavejester/codox/pull/137

Also, you can take a look at the [live documentation of my gadjett library](http://viebel.github.io/gadjett/gadjett.collections.html#var-map-object)

**Comment by expez** [comment]
Hot damn, that's awesome!

**Comment by viebel** [comment]
Yeah. I was surprised by how easy it was to integrate KLIPSE with Codox.

Codox really rocks!

Go ahead! Give it a try and make your own live doc!
